packages/g/geany-themes/pspec_x86_64.xml

```diff
@@ -77,7 +78,10 @@
 </Description>
         <PartOf>editor</PartOf>
         <Files>
-            <Path fileType="doc">/usr/share/doc</Path>
+            <Path fileType="doc">/usr/share/doc/geany-themes/AUTHORS</Path>
```

Comment:
It is not necessary to include `doc` folder. You can remove it by adding a new line in `install :` step:
`rm -rf $installdir/usr/share/doc`